backstop_data/engine_scripts/puppet/dashboard/configure-dashboard.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = async (engine) => {
+  await engine.click('#crm-dashboard-configure');
+  await engine.waitFor('#configure-dashlet', { visible: true });
+  await engine._client.send('Animation.setPlaybackRate', { playbackRate: 2 })
```

Comment:
what's this? what does it do?

Reply:
So, basically when we click on *Configure dashboard* the edit dashboard screen opens with a fade in animation. So the screenshot captured some times doesn't match as they are translucent (fade in effect not completed at time of screenshot).
So this code virtually fast forward the screenshot process (by increasing the playback Rate) after the animation.
Link to reference - https://github.com/GoogleChrome/puppeteer/issues/453